ui/base-tool/src/main/res/layout/tract_settings_sheet.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
why did you copy `tract_settings_sheet` here, and is it different than `tool_option_settings_sheet`?